tests/Commands/QueueFailedTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Commands;
+
+use CodeIgniter\I18n\Time;
+use CodeIgniter\Queue\Models\QueueJobFailedModel;
+use CodeIgniter\Test\Filters\CITestStreamFilter;
+use Exception;
+use Tests\Support\CLITestCase;
+
+/**
+ * @internal
+ */
+final class QueueFailedTest extends CLITestCase
+{
+    /**
+     * @throws Exception
+     */
+    public function testRun(): void
+    {
+        Time::setTestNow('2023-12-19 14:15:16');
```

Comment:
> Don’t forget to reset the current time after the test with calling it without parameters.
https://codeigniter4.github.io/CodeIgniter4/testing/overview.html#testing-and-time

Reply:
Thank you.